icu4c/source/data/misc/supplementalData.txt

```diff
@@ -35597,398 +35597,385 @@ supplementalData:table(nofallback){
-            length-visiblty-small{"foot"}
-            mass-person{"pound"}
-            mass-person-small{"pound ounce"}
-            pressure-baromtrc{"inch-hg"}
-            speed-road-travel{"mile-per-hour"}
-            speed-wind{"mile-per-hour"}
-            temperature-person{"fahrenheit"}
-            temperature-weather{"fahrenheit"}
-            volume-vehicle-fuel{"gallon"}
+        "duration"{
+            "music-track"{
+                001{
+                    0{
+                        unit{"minute+second"}
+                    }
+                    1{
+                        unit{"second"}
+                    }
+                }
+            }
+            "person-age"{
+                001{
+                    0{
+                        unit{"year-person+month-person"}
+                    }
+                    1{
+                        unit{"month-person"}
+                    }
+                }
+            }
+            "tv-program"{
+                001{
+                    0{
+                        unit{"minute+second"}
+                    }
+                    1{
+                        unit{"second"}
+                    }
+                }
+            }
+            _default{
+                001{
+                    0{
+                        unit{"year"}
+                    }
+                    1{
+                        unit{"month"}
+                    }
+                    2{
+                        unit{"day"}
+                    }
+                    3{
+                        unit{"hour"}
+                    }
+                    4{
+                        unit{"minute"}
+                    }
+                    5{
+                        unit{"second"}
+                    }
+                    6{
+                        unit{"millisecond"}
+                    }
+                    7{
+                        unit{"microsecond"}
+                    }
+                    8{
+                        unit{"nanosecond"}
+                    }
+                }
+            }
         }
-        VC{
-            concentr-blood-glucose{"millimole-per-liter"}
+        // DEMO: alt{"informal"}:
+        "length"{
+            "road"{
+                001{
+                    0{
+                        unit{"kilometer"}
+                    }
+                }
+                GB{
+                    0{
+                        unit{"mile"}
+                    }
+                }
+                SE{
+                    0{
+                        unit{"mile-scandinavian"}
+                        alt{"informal"}
+                    }
+                }
+                US{
+                    0{
+                        unit{"mile"}
+                    }
+                }
+                // DEMO: Current hack for _small, proposing we drop this info and simply use geq:
+                _small{
+                    001{
+                        0{
```

Comment:
Can you make this an array instead of a 0/1/2/... table?

Reply:
How does this look to you? This was actually generated by <FIFO> rules. And there are additional changes pulled in which I didn't manually remove again - makes it easier to regenerate. (Output is generated on a cldr branch that's based on sffc/units-staging.)